tools/util/getFolderLetter.ts

```diff
@@ -0,0 +1,7 @@
+export default function getFolderLetter(service: string) {
```

Comment:
are we seriously dedicating a whole file for a 5 line function? juse use a util file as we had before

Reply:
It's more organized like that than having it in one big file with util functions. And the implementation will change with App support anyway.

Reply:
Florian, there are only **three** functions which are never maintained as if they work now, most likely they'll work for a while. There is no issue with creating one file for all of these.

Reply:
Tbh I am for the utility-function gets it's own file syntax. If it was only used in 1 other file it should just be in that file, otherwise it should get it's own.

Reply:
The functions we have here are 'light functions' which don't need a high maintenance need as other functions need. Once this PR is merged, it probably won't get touched again until it is rewritten again. I can understand having a file per function **if** there were many functions and they are frequently updated, but that just doesn't happen on the presence repo

Reply:
Hmm I personally don't agree with that, I get that they are light functions that dont require much maintenance, but still IMO it's better practice this way, so you only import the function you NEED and not a whole file which may contain functions you dont need.

Reply:
Turning away from what __you__ may believe is best practice and thinking about industry standards, importing a `util.ts` file is used more widely when you only have three functions. Also looking at this snippet, the three `tools/auto`, they use two of three functions and I'm fairly sure it will be more performant to import the file instead of importing two separate files, despite importing a function you may not need. 

`presenceValidator` uses `getDiff`
`presenceUpdater` uses `getDiff` `getFolderLetter`
`compileChanged` uses `getLatestSchema` and `getDiff`